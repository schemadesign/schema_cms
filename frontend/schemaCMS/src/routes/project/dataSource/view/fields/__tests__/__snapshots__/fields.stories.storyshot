// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Project/DataSource/View/Fields Default 1`] = `
<div
  className="sc-ksYbfQ jHwjmi"
>
  <div
    className="sc-hqyNC keRCOE"
  >
    <div
      className="sc-hmzhuo gPZAdw"
    >
      <div
        className="sc-frDJqD eoHuEa"
      >
        <button
          disabled={true}
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#f0efef",
              "border": "none",
              "borderRadius": "48px",
              "color": "#1d1d20",
              "cursor": "pointer",
              "display": "inline-block",
              "fontSize": "18px",
              "height": 60,
              "margin": "0",
              "minHeight": "48px",
              "outline": "none",
              "padding": "0",
              "width": 60,
            }
          }
        >
          <svg
            style={
              Object {
                "fill": "#1d1d20",
                "height": "60px",
                "opacity": 0.3,
                "width": "60px",
              }
            }
            viewBox="0 0 60 60"
          >
            <path
              d="M25.857 17L28 19.083 18.828 28H45v3H18.828L28 39.917 25.857 42 13 29.5 25.857 17z"
            />
          </svg>
        </button>
      </div>
      <div
        className="sc-kvZOFW geyuTt"
      >
        {countFields} {countFields, plural,one {Field} other {Fields}}
      </div>
      <div
        className="sc-frDJqD eoHuEa"
      >
        <button
          disabled={false}
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#f0efef",
              "border": "none",
              "borderRadius": "48px",
              "color": "#1d1d20",
              "cursor": "pointer",
              "display": "inline-block",
              "fontSize": "18px",
              "height": 60,
              "margin": "0",
              "minHeight": "48px",
              "outline": "none",
              "padding": "0",
              "width": 60,
            }
          }
        >
          <svg
            style={
              Object {
                "fill": "#1d1d20",
                "height": "60px",
                "width": "60px",
              }
            }
            viewBox="0 0 60 60"
          >
            <path
              d="M34.143 17L32 19.083 41.172 28H15v3h26.172L32 39.917 34.143 42 47 29.5 34.143 17z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      className="sc-fAjcbJ kLEWEn"
    >
      <table
        className="sc-caSCKo dFAagl"
      >
        <thead
          className="sc-gisBJw kDlVKO"
        >
          <tr
            className="sc-cHGsZl kZpRKm"
          >
            <th
              className="sc-kgAjT kVzWDK"
            >
              #
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              First Name
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              Last Name
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              Full Name
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              Birthplace
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              State of Birth
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              Birthday
            </th>
            <th
              className="sc-kgAjT kVzWDK"
            >
              Url
            </th>
          </tr>
        </thead>
        <tbody
          className="sc-kjoXOD hpTAeq"
        >
          <tr
            className="sc-cHGsZl kZpRKm"
          >
            <td
              className="sc-cJSrbW gtNobl"
            >
              1
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              George
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Washington
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              George Washington
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Westmoreland County
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              VA
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              02/22/1732
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cHGsZl kZpRKm"
          >
            <td
              className="sc-cJSrbW gtNobl"
            >
              2
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              John
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Adams
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              John Adams
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Braintree
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              MA
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              10/30/1735
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cHGsZl kZpRKm"
          >
            <td
              className="sc-cJSrbW gtNobl"
            >
              3
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Thomas
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Jefferson
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Thomas Jefferson
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Shadwell
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              VA
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              04/13/1743
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cHGsZl kZpRKm"
          >
            <td
              className="sc-cJSrbW gtNobl"
            >
              4
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              James
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Madison
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              James Madison
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Port Conway
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              VA
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              03/16/1751
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cHGsZl kZpRKm"
          >
            <td
              className="sc-cJSrbW gtNobl"
            >
              5
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              James
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Monroe
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              James Monroe
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              Monroe Hall
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              VA
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              04/28/1758
            </td>
            <td
              className="sc-TOsTZ eLxLAK"
            >
              some.url.com
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
