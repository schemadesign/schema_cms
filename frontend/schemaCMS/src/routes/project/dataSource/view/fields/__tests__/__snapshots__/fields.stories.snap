// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Project/DataSource/View/Fields Default 1`] = `
<div
  className="sc-jlyJG bbyJzT"
>
  <div
    className="sc-bRBYWo ccSRck"
  >
    <div
      className="sc-gipzik eFNLgT"
    >
      <div
        className="sc-csuQGl fWqAYk"
      >
        <button
          disabled={true}
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#f0efef",
              "border": "none",
              "borderRadius": "48px",
              "color": "#1d1d20",
              "cursor": "pointer",
              "display": "inline-block",
              "fontSize": "18px",
              "height": 60,
              "margin": "0",
              "minHeight": "48px",
              "outline": "none",
              "padding": "0",
              "width": 60,
            }
          }
        >
          <svg
            style={
              Object {
                "fill": "#1d1d20",
                "height": "60px",
                "width": "60px",
              }
            }
            viewBox="0 0 60 60"
          >
            <path
              d="M25.857 17L28 19.083 18.828 28H45v3H18.828L28 39.917 25.857 42 13 29.5 25.857 17z"
            />
          </svg>
        </button>
      </div>
      <div
        className="sc-Rmtcm cXseFN"
      >
        {countFields} {countFields, plural,one {Field} other {Fields}}
      </div>
      <div
        className="sc-csuQGl fWqAYk"
      >
        <button
          disabled={false}
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#f0efef",
              "border": "none",
              "borderRadius": "48px",
              "color": "#1d1d20",
              "cursor": "pointer",
              "display": "inline-block",
              "fontSize": "18px",
              "height": 60,
              "margin": "0",
              "minHeight": "48px",
              "outline": "none",
              "padding": "0",
              "width": 60,
            }
          }
        >
          <svg
            style={
              Object {
                "fill": "#1d1d20",
                "height": "60px",
                "width": "60px",
              }
            }
            viewBox="0 0 60 60"
          >
            <path
              d="M34.143 17L32 19.083 41.172 28H15v3h26.172L32 39.917 34.143 42 47 29.5 34.143 17z"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      className="sc-eHgmQL fUBxPx"
    >
      <table
        className="sc-cvbbAY jfCLrQ"
      >
        <thead
          className="sc-jWBwVP fDrbnp"
        >
          <tr
            className="sc-cMljjf lfxHVl"
          >
            <th
              className="sc-jDwBTQ khizas"
            >
              #
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              First Name
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              Last Name
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              Full Name
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              Birthplace
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              State of Birth
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              Birthday
            </th>
            <th
              className="sc-jDwBTQ khizas"
            >
              Url
            </th>
          </tr>
        </thead>
        <tbody
          className="sc-brqgnP SqPLV"
        >
          <tr
            className="sc-cMljjf lfxHVl"
          >
            <td
              className="sc-gPEVay gLBXLW"
            >
              1
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              George
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Washington
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              George Washington
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Westmoreland County
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              VA
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              02/22/1732
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cMljjf lfxHVl"
          >
            <td
              className="sc-gPEVay gLBXLW"
            >
              2
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              John
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Adams
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              John Adams
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Braintree
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              MA
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              10/30/1735
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cMljjf lfxHVl"
          >
            <td
              className="sc-gPEVay gLBXLW"
            >
              3
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Thomas
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Jefferson
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Thomas Jefferson
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Shadwell
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              VA
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              04/13/1743
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cMljjf lfxHVl"
          >
            <td
              className="sc-gPEVay gLBXLW"
            >
              4
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              James
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Madison
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              James Madison
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Port Conway
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              VA
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              03/16/1751
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              some.url.com
            </td>
          </tr>
          <tr
            className="sc-cMljjf lfxHVl"
          >
            <td
              className="sc-gPEVay gLBXLW"
            >
              5
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              James
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Monroe
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              James Monroe
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              Monroe Hall
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              VA
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              04/28/1758
            </td>
            <td
              className="sc-jAaTju cdDUMW"
            >
              some.url.com
            </td>
          </tr>
        </tbody>
      </table>
    </div>
  </div>
</div>
`;
